projects/fal/src/fal/toolkit/file/file.py

```diff
@@ -135,19 +135,24 @@ def from_bytes(
             FileRepository | RepositoryId
         ] = FALLBACK_REPOSITORY,
         request: Optional[Request] = None,
+        save_kwargs: Optional[dict] = None,
+        fallback_save_kwargs: Optional[dict] = None,
```

Comment:
not a fan of this approach but needed a way to pass down the params specific to the repo itself (s3) that are not arguments of the base class.